omise/api/resources/charge.py

```diff
@@ -0,0 +1,170 @@
+import copy
```

Comment:
@vernomcrp copy only uses in `card.py`, I saw you import it in many files, I think we can remove it.